containers/NFTMarketplace/NFTItem/index.js

```diff
@@ -75,7 +86,7 @@ const NFTItem = ({
         <Grid item xs={12}>
           <ListItem
             title='Minted'
-            value={`${nft.minted || 0} / ${nft.max || 0}`}
+            value={`${nft.supply || 0} / ${nft.max || 0}`}
```

Comment:
seems to repeat the above file?

Reply:
We can see NFT's minted Value on item and detail modal. So this field is repeated.
![image](https://user-images.githubusercontent.com/45635459/126117823-8c8ca693-4955-48cc-9ff2-8f0e0bfb6080.png)